Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG GOLANG_BUILDER=registry.access.redhat.com/ubi9/go-toolset:1.21
+ARG GOLANG_BUILDER=registry.access.redhat.com/ubi9/go-toolset:1.24
 ARG OPERATOR_BASE_IMAGE=registry.access.redhat.com/ubi9/ubi-minimal:latest
```

Comment:
We may want to consider setting the new FIPS env variable GOFIPTS14=true to do the extra build time checks here

Reply:
https://go.dev/doc/security/fips140

Reply:
as discussed on slack, we'll do that in a follow up